roles/keycloak_quarkus/tasks/restart/serial_then_parallel.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Verify first restarted service with health URL, then rest restart in parallel
+  block:
+    - name: "Restart and enable {{ keycloak.service_name }} service on first host"
```

Comment:
we should re-use `restart.yml` here with `delegate_to: "{{ ansible_play_hosts | first }}"` and `run_once: true`, i.e. replacing the next three tasks with this include instead

Reply:
not really; restart.yml can have the health check turned off, while in there is enforced

Reply:
your right, but still: e.g. introducing `keycloak_quarkus_internal_enforce_health_check` and setting it to true here would lower the maintenance burden on the long run -- your call, of course

Reply:
mmh after second thought, yeah you're right, let me amend that